app/git/utils.py

```diff
@@ -590,11 +591,45 @@ def get_interested_actions(github_url, username, email=''):
     return actions_by_interested_party
 
 
-def get_user(user, sub_path=''):
+def get_user(user, sub_path='', scope='', auth=_AUTH):
```

Comment:
_AUTH is the GITHUB_API_TOKEN (application token) but need to be the profile.access_token and the request:
`https://api.github.com/user/orgs?access_token=XXXX` or by header like `get_github_primary_email`
```
headers = dict({'Authorization': f'token {oauth_token}'}, **JSON_HEADER)
response = requests.get('https://api.github.com/user/emails', headers=headers)
```

Reply:
I will add you to an org so you to an org for testing "whatcomesnext" so you can reproduce 
yo need to set up it as private (so no visible on you profile for others) 
![image](https://user-images.githubusercontent.com/1358093/65934399-96e63600-e3eb-11e9-8383-57bf86afc350.png)